docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,269 @@
+# DALi Widget Application
+
+
+You can create a widget application with DALi to display small version of application on the Home screen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> - The implementation of DALi widget application is similar to [DALi basic UI application](dali-basic-app.md), because the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class.
+  > - Ensure you are familiar with the basic UI application details before tackling a widget application.
+
+- Ensure you start the event loop.
+
+- To implement your own widget class inherit the `Dali::Widget` class.
+
+![DALi widget application](./media/dali_widget_application.png)
+
+**Figure: DALi widget application**
+
+For more information on basics of creating a DALi widget application, see [Create a DALi Widget Application](#create).
```

Comment:
@sabeena21 For this kind of "For more inforamtion ~", how about thinking to make a not-broken sentence even there is no link?

For example, in [basic app page](https://github.com/Samsung/tizen-docs/blob/ee66c577310ab05f12f6681c88cceb8b7d4e7733/docs/application/native/guides/app-management/dali-basic-app.md), "For more information on Tizen UI applications in general, see UI Application." it self makes a not-broken sentence. 

In this PR, "For more information on basics of creating a DALi widget application, see Create a DALi Widget Application" is weird if we don't consider the hyper link.

Reply:
@mijin85cho,  It is good to consider a hyperlink.

Reply:
@sabeena21 I'm suggesting you to polish that sentence, because it is breaking consistency. If you search other "For more information, see " sentences in guide pages, you would see these sentences are ended with noun phrase, making "For more information, see Noun" form.

Reply:
@sabeena21 
As the focus of that section is "creating", how about 

For more information on basics of creating a DALi widget application, see [Creating a DALi Widget Application](#create).

I can't help but feel something lacking as the sentence and the title are different, by the way..

Reply:
@mijin85cho, as far as keeping the hyperlink similar to the title.
The suggested sentence by you is good to be considered.

Reply:
@sabeena21 Thanks for the check. 

@neostom432 Please update the sentence as to "For more information on basics of creating a DALi widget application, see [Creating a DALi Widget Application](#create)."

Reply:
@mijin85cho done